documentation/src/main/asciidoc/topics/con_xsite_site_masters.adoc

```diff
@@ -0,0 +1,11 @@
+[id='xsite_masters-{context}']
+= Site Masters
+Site masters are nodes in {brandname} clusters that use the JGroups RELAY2
```

Comment:
maybe remove the reference to RELAY2 and keep it simple: 
Site masters are the nodes responsible to send and receive requests from backup locations. 
If a node isn't a SiteMaster and needs to send a request to a backup location, it needs to find a SiteMaster in the local cluster to forward the request.

Reply:
sounds good. thanks.